### PR TITLE
added description and installation guides to README

### PR DESCRIPTION
**Pull request by swetshaw**
Created a new PR for #2 since the old PR #6 had merge conflicts. We can scrap PR #6  and take this PR into consideration instead.

### CODE REVIEW

**Review by etr2460 — COMMENTED** [review]
one main comment here, i think typically the README.md is built for consumers of the package, while the CONTRIBUTING.md is for people who want to contribute in open source. so i'd recommend removing the storybook instructions and instead start documenting how to use the package in the readme. we can always keep the contributing CTA at the bottom which will include instructions on how to develop in the repo

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+web3-ui is a library made up of UI components specifically crafted for web3 use cases.
 
 [Read the RFC](https://forum.developerdao.com/t/rfc-web3-ui-a-web3-specific-ui-library)
 
+## Installation
+
+web3-ui is available as an [npm package](https://www.npmjs.com/package/web3-ui)
+
+Install using:
+```npm install web3-ui```
```
Comment (codingwithmanny) [inline README.md:12]:
```suggestion
Install using:
```bash
npm install web3-ui
```
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+ or
+```npm i web3-ui```
+or
+```yarn add web3-ui```
```
Comment (codingwithmanny) [inline README.md:16]:
```suggestion
```bash
yarn add web3-ui;
# npm i web3-ui;
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+
+To start storybook and see the components already created, use:
+```yarn storybook```
```
Comment (codingwithmanny) [inline README.md:19]:
```suggestion
```bash
yarn storybook
```
```

**Comment by codingwithmanny** [comment]
@swetshaw Added a couple suggestions for the code changes for the formatting for the markdown.
A small trick is to run prettier on the markdown file to see if it auto formats the markdown correctly with:

```bash
npx prettier --write README.md;
```

File: README.md (hunk 3 above)
Comment (swetshaw) [reply README.md]: I have removed the instruction for how to run storybook because I found @etr2460 comment above valid.

File: README.md (hunk 2 above)
Comment (swetshaw) [reply README.md]: Hi I have updated the README based on this suggestion.

**Comment by swetshaw** [comment]
> @swetshaw Added a couple suggestions for the code changes for the formatting for the markdown. A small trick is to run prettier on the markdown file to see if it auto formats the markdown correctly with:
> 
> ```shell
> npx prettier --write README.md;
> ```

Thanks for the suggestion, it was really useful. Used that. 👍

File: README.md [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
+
+web3-ui is available as an [npm package](https://www.npmjs.com/package/web3-ui)
+
+Install using:
+
+```bash
+yarn add web3-ui;
+or
+npm i web3-ui;
+```
```
Comment (etr2460) [inline README.md:17]: @Dhaiwat10 can you verify if these are correct, or did we move the packages to the @web3-ui namespace

Comment (Dhaiwat10) [reply README.md:17]: We haven't actually published the components package yet. Will do that once some components are ready and ping here.

**Comment by Dhaiwat10** [comment]
Thanks for this @swetshaw I'll drop an update once we roll out the first versions of the packages.

**Comment by Dhaiwat10** [comment]
I think we unfortunately might need to close this since we kinda diverged away from this initial idea and this PR became stale :( 

Thanks though!

**Pull request closed without merging**